website/views.py

```diff
@@ -6,6 +6,7 @@
 from django.contrib.auth import authenticate, login, logout
 from django.contrib.auth.models import User
 from django.contrib.auth.decorators import login_required
+from django.contrib.auth.mixins import LoginRequiredMixin
```

Comment:
I like the use of this mixin. Can we make a similar mixin, that extends this one, but also ensures the user has a verified email address?  These views will need both logged in, and verified.